googleapis/google-cloud-node
# migrate code from googleapis/nodejs-iap

- feat: initial stub of library
- feat!: initial generation of library (#1)
- chore: release 1.0.0 (#2)
- chore(deps): update dependency sinon to v11 (#4)
- chore(nodejs): update client ref docs link in metadata (#5)
- fix(deps): google-gax v2.24.1 (#7)
- chore: release 1.0.1 (#8)
- feat: turns on self-signed JWT feature flag (#9)
- chore: release 1.1.0 (#10)
- chore: disable renovate dependency dashboard (#1194) (#11)
- fix(build): migrate to main branch (#12)
- chore: release 1.1.1 (#13)
- docs(samples): add auto-generated Node samples (#17)
- build(node): update deps used during postprocessing (#1243) (#20)
- chore: fix the wrong post processor image (#22)
- chore(deps): update dependency @types/node to v16 (#21)
- chore(deps): update dependency sinon to v12 (#23)
- docs(samples): add example tags to generated samples (#24)
- build: add generated samples to .eslintignore (#25)
- docs(node): support "stable"/"preview" release level (#1312) (#28)
- chore: add api_shortname and library_type to repo metadata (#27)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#30)
- test(nodejs): remove 15 add 16 (#1322) (#33)
- build: update copyright year to 2022 (#35)
- chore: update v2.12.0 gapic-generator-typescript (#37)
- chore(deps): update dependency sinon to v13 (#39)
- docs(nodejs): version support policy edits (#1346) (#41)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#42)
- chore: update v2.14.2 gapic-generator-typescript (#46)
- feat: AuditConfig for IAM v1 (#48)
- chore(main): release 1.2.0 (#49)
- build(node): update client library version in samples metadata (#1356) (#57)
- chore(deps): update dependency @types/mocha to v9 (#58)
- chore(deps): update dependency sinon to v14 (#61)
- feat: add the TunnelDestGroup-related methods and types (#63)
- build!: update library to use Node 12 (#65)
- chore(main): release 2.0.0 (#64)
- chore(deps): update dependency jsdoc-fresh to v2 (#67)
- chore(deps): update dependency jsdoc-region-tag to v2 (#68)
- fix(docs): document fallback rest option (#69)
- chore(deps): update dependency linkinator to v4 (#72)
- chore(main): release 2.0.1 (#70)
- fix: better support for fallback mode (#73)
- fix: change import long to require (#74)
- fix: do not import the whole google-gax from proto JS (#1553) (#77)
- fix: allow passing gax instance to client constructor (#78)
- chore(main): release 2.0.2 (#75)
- fix: preserve default values in x-goog-request-params header (#79)
- test: use fully qualified request type name in tests (#82)
- fix(deps): use google-gax v3.5.2 (#86)
- chore(deps): update dependency @types/node to v18 (#84)
- chore(deps): update dependency jsdoc to v4 (#88)
- fix: regenerated protos JS and TS definitions (#89)
- chore(main): release 2.0.3 (#80)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
